Mathlib/Analysis/Complex/Hadamard.lean

```diff
@@ -393,12 +397,13 @@ lemma norm_le_interp_of_mem_verticalClosedStrip' (f : ℂ → E) {z : ℂ} {a b
       rcases h with ⟨h0, h1⟩
       rw [ne_comm] at h0 h1
       simp_rw [interpStrip_eq_of_pos f _ (lt_of_le_of_ne (sSupNormIm_nonneg f 0) h0)
-        (lt_of_le_of_ne (sSupNormIm_nonneg f 1) h1)]
+      (lt_of_le_of_ne (sSupNormIm_nonneg f 1) h1)]
       simp only [norm_eq_abs, map_mul]
       rw [abs_cpow_eq_rpow_re_of_pos ((Ne.le_iff_lt h0).mp (sSupNormIm_nonneg f _)) _]
       rw [abs_cpow_eq_rpow_re_of_pos ((Ne.le_iff_lt h1).mp (sSupNormIm_nonneg f _)) _]
       simp only [sub_re, one_re, le_refl]
-  apply (norm_le_interpStrip_of_mem_verticalClosedStrip f hz hd hB).trans (this.trans _)
+
```

Comment:
Why a blank line? If you feel like this is a nice spot to semantically split the proof, then you should add a comment, not a blank line
```suggestion
```